serokell-style.md

```diff
@@ -233,49 +240,58 @@ align by max width among them.
 
 Use singular when naming modules (e.g. use `Data.Map` and
```

Comment:
I still don't find this paragraph useful, but whatever.
Don't do it, but sometimes do it :man_shrugging: